core.php

```diff
@@ -504,23 +504,23 @@
     /*
      * 3.6.0
      */
-    "results_per_page"          => "Results Per Page",
-    "class_name"                => "Class Name",
-    "possible_permission_issue" => "You are running the command as \":cli_user\", while \":path\" is owned by \":path_user\", this can lead to file permission issues.",
-    "system_update"             => "System Update",
-    "system_update_desc"        => "Update SupportPal to the latest version available.",
-    "update_available"          => "Update Available",
+    "results_per_page"          => "Resultaten per pagina",
+    "class_name"                => "Klasse Naam",
+    "possible_permission_issue" => "U voert de opdracht uit als  \":cli_user\", terwijl \":path\" is eigendom van \":path_user\", dit kan leiden tot problemen met bestandsrechten.",
```

Comment:
@TimVNL correct me if I'm wrong but I think it should be:

```suggestion
    "possible_permission_issue" => "U voert de opdracht uit als \":cli_user\", terwijl \":path\" eigendom is van \":path_user\", dit kan leiden tot problemen met bestandsrechten.",
```